modules/cloudsql/main.tf

```diff
@@ -21,22 +21,22 @@ locals {
   random_hash     = var.suffix
   cloudsql_name   = "forseti-server-db-${local.random_hash}"
   network_project = var.network_project != "" ? var.network_project : var.project_id
+  cloudsql_zone   = "${var.cloudsql_region}-c"
```

Comment:
This is the only functional change. Everything else is some cleanup of variable definitions and updated terraform syntax.